src/model/paths.ml

```diff
@@ -39,7 +39,7 @@ module Identifier = struct
   let name : [< t] -> string = function
     | `Root(_, name) -> UnitName.to_string name
     | `Page(_, name) -> PageName.to_string name
-    | `Module(_, name) -> ModuleName.to_string name
+    | `Module(_, name) -> ModuleName.to_string_unsafe name
     | `Argument(_, _, name) -> ArgumentName.to_string name
     | `ModuleType(_, name) -> ModuleTypeName.to_string name
     | `Type(_, name) -> TypeName.to_string name
```

Comment:
`ModuleName`, `ArgumentName` and `TypeName` are all of type `Name`, but you use `to_string_unsafe` only for the first one? Why is that?

Reply:
Ah! good catch. None of them should be using unsafe.